new_relic_scripts/sms_sign_in.js

```diff
@@ -0,0 +1,44 @@
+require('./bootstrap');
+
+const {
+  fillOutOTPForm,
+  fillOutSignInForm,
+  submitForm,
+  verifyCurrentPath,
+  visitURL,
+} = require('./support/browser');
+const { getOTPFromTwilio } = require('./support/twilio_helpers');
+
+const {
+  IDP_URL,
+  NR_SMS_SIGN_IN_EMAIL,
+  NR_SMS_SIGN_IN_PASSWORD,
+  NR_SMS_SIGN_IN_TWILIO_PHONE
+} = process.env;
+
+const otpSentAfter = new Date();
```

Comment:
should we move this into the step where it gets used? or do we want this to be the start of the whole script?